### ISSUE TITLE
Should I change learning rate or max_batch when I use multi-GPU instead of single-GPU?

### ISSUE THREAD

**Issue by lsydd** [AlexeyAB/darknet#7687]
In yolov3.coco-giou-12.cfg, I see for different GPU num, the learning rate in cfg is changed.

```
## single gpu
learning_rate=0.001
burn_in=1000
max_batches = 550400

## 2 gpu
#learning_rate=0.0005
#burn_in=2000
#max_batches = 500200

## 4 gpu
#learning_rate=0.00025
#burn_in=4000
#max_batches = 500200
###max_batches = 2000800
```

Would you explain the mechanism of multi-GPU training in Darknet and the effect of using it? 

Thank you for help.

**Comment by stephanecharette** [comment]
https://github.com/AlexeyAB/darknet#how-to-train-with-multi-gpu
